samples/host_verify/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.11)
+
+project("Remote Host-side Enclave Verification Sample" LANGUAGES C CXX)
+
+find_package(OpenEnclave CONFIG REQUIRED)
+
+set(CMAKE_CXX_STANDARD 11)
+
+add_executable(host_verify host.c)
+
+target_include_directories(host_verify PRIVATE)
+
+target_link_libraries(host_verify openenclave::oehostverify)
+
+add_custom_target(run
+  DEPENDS host_verify
+  COMMAND openssl ecparam -name prime256v1 -genkey -noout -out keyec.pem
+  COMMAND openssl ec -in keyec.pem -pubout -out publicec.pem
+  COMMAND openssl genrsa -out keyrsa.pem 2048
+  COMMAND openssl rsa -in keyrsa.pem -outform PEM -pubout -out publicrsa.pem
+  COMMAND pwd
+  COMMAND ../../tests/tools/oecert/host/oecert ../../tests/tools/oecert/enc/oecert_enc --cert keyec.pem publicec.pem --out sgx_cert_ec.der
+  COMMAND ../../tests/tools/oecert/host/oecert ../../tests/tools/oecert/enc/oecert_enc --cert keyrsa.pem publicrsa.pem --out sgx_cert_rsa.der
+  COMMAND ../../tests/tools/oecert/host/oecert ../../tests/tools/oecert/enc/oecert_enc --report --out sgx_report.bin
+  COMMAND host_verify -c sgx_cert_ec.der -c sgx_cert_rsa.der -r sgx_report.bin)
```

Comment:
Consider this sample being installed as part of the host_verify package. There will be no oecert. This section needs to be moved to somewhere that only runs during CI/CD.

Reply:
This would only be run if you do a `make run`. You'll still be able to run a normal build. Maybe we could output a warning message that this target will only run on an SGX-enabled machine? What do you think?

Reply:
oecert is a test tool which is not included in any package. When the user install host_verify package, this sample will show up in /opt/openenclave/share/openenclave/samples. make run from there will fail.

I suggest we move the oecert section from CMakeLists.txt/Makefile to Jenkinsfile just for CI/CD verification.

Reply:
The jenkinsfiles just kick off stuff defined in cmake. But I see your point. We can't include it in the sample if it's not a published tool.

I think there may be value in publishing the tool (mostly for people to use in development and testing of attestation clients and services).

Reply:
I opened https://github.com/openenclave/openenclave/issues/2898 to track this suggestion. Providing this tool makes it easier for developers to use this sample and understand the format of the data that should be passed to it.

Reply:
In the latest update I comment out the lines related to oecert and there's no problem running it. Instead, if the files are already generated, then run the verification; otherwise, display the usage.

In a discussion with Yen this afternoon I realized that there's a pull request that may be related to both this issue and CI/CD: #2519. Since the commands are identical as those in the jenkins files, if #2519 can be resolved, then oecert does not have to be included in the sample and it can be part of the routine. What do you think?